internal/cmd/helm-operator/run/cmd.go

```diff
@@ -87,9 +89,10 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 	// Load config options from the config at f.ManagerConfigPath.
 	// These options will not override those set by flags.
 	var (
-		options manager.Options
+		options = ctrl.Options{Scheme: scheme}
```

Comment:
if that is an issue for Ansible, my 2cents that also happens for Helm.
But we need to check. We do not have e2e for the run command. Maybe by adding the tests we could ensure and avoid it in the future. If you decide to do the tests are implemented in : https://github.com/operator-framework/operator-sdk/tree/master/test/e2e (see e.g: https://github.com/operator-framework/operator-sdk/blob/master/test/e2e/go/local_test.go )